packages/models/organizations.coffee

```diff
@@ -4,6 +4,19 @@ Organization = Astro.Class
   collection: Organizations
   transform: true
   fields:
-    name: 'string'
-    description: 'string'
-    createdById: 'string'
+    name:
+      type: 'string'
+      validators: [
+        Validators.unique()
+        Validators.string()
+      ]
+    description:
+      type: 'string'
+      validators: [
+        Validators.string()
+      ]
+    createdById:
+      type: 'string'
+      validators: [
+        Validators.string()
```

Comment:
That gets tricky, and I'm not sure it's exactly what we want - if we ever need to delete a user, we probably don't want their organizations to automatically be invalid.  Since we set the createdById in the controller and we have a test that makes sure users can access organizations they've created, I don't think we need more validation there.  We may want to add a Required validation just in case though.